pkg/cassandra/config.go

```diff
@@ -26,11 +27,23 @@ type config struct {
 	additionalJvmOptions []string
 }
 
+type ServerEncryptionOptionsYaml struct {
+	encryption.ServerEncryptionOptions `json:",inline"`
+	encryption.EncryptionStoresYaml    `json:",inline"`
+}
+
+type ClientEncryptionOptionsYaml struct {
+	encryption.ClientEncryptionOptions `json:",inline"`
+	encryption.EncryptionStoresYaml    `json:",inline"`
+}
+
 // CassYamlIR is an internal representation of the cassandra.yaml. It is required because we want to make some options (esp. start_rpc) invisible to the user,
 // but some of those options still need to be rendered out into the final cassandra.yaml.
 type CassYamlIR struct {
```

Comment:
Just FYI. I see that this type has been added recently. This file is becoming a bit messy and would need a bit of tidying. E.g. this type should not be exported. Also, the `config` type above was already meant as the "internal representation of the cassandra.yaml", so now we have a bit of a duplication here. But we can look into it in another PR. Just something to keep in mind.